crates/cli/src/subcommands/publish.rs

```diff
@@ -101,6 +102,20 @@ pub fn cli() -> clap::Command {
                 .short('s')
                 .help("The nickname, domain name or URL of the server to host the database."),
         )
+        .arg(
+            Arg::new("force")
+                .long("force")
+                .action(SetTrue)
+                .help("DANGEROUS - Proceed with all actions without waiting for user confirmation")
+        )
+        .arg(
+            Arg::new("deprecated_clear_database")
+                .long("clear-database")
+                .requires("name_or_address")
+                .conflicts_with("destroy_previous")
+                .action(SetTrue)
+                .help("DEPRECATED - Use --destroy-previous"),
```

Comment:
Why did we add this?

Reply:
I see this is a backwards compatible thing - let's just mark this as an API break and merge it for 0.9.